xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_dll.h

```diff
@@ -115,6 +115,16 @@ extern "C"
    * @remarks Required if add-on supports playing epg tags. Return PVR_ERROR_NOT_IMPLEMENTED if this add-on won't provide this function.
    */
   PVR_ERROR IsEPGTagPlayable(const EPG_TAG* tag, bool* bIsPlayable);
+  
+  /*!
+  * Retrieve the edit decision list (EDL) of an EPG tag on the backend.
+  * @param epgTag The EPG tag.
+  * @param edl out: The function has to write the EDL list into this array.
+  * @param size in: The maximum size of the EDL, out: the actual size of the EDL.
+  * @return PVR_ERROR_NO_ERROR if the EDL was successfully read.
+  * @remarks Required if bSupportsEpgEdl is set to true. Return PVR_ERROR_NOT_IMPLEMENTED if this add-on won't provide this function.
+  */
+  PVR_ERROR GetEPGTagEdl(const EPG_TAG& epgTag, PVR_EDL_ENTRY edl[], int *size);
```

Comment:
As this is a C API, C++ language consructs like references are not okay here. Pleaes use a pointer. And yes, I know you copied GetRecordingEdl - but we should not repeat the mistakes we did in the past.